backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/BPPrintFormatQuery.java

```diff
@@ -37,36 +43,27 @@
  *
  */
 
+@Getter
 @Value
-@Builder
+@Builder(toBuilder = true)
+@RequiredArgsConstructor
```

Comment:
(don't have to change it only for this comment)

FYI, 
1. if u annotate with `@Value` you don't have to annotate with `@Getter` because Value includes Getter.

2. Don't see the point of annotating with RequiredArgsConstructor when annotating with Builder

Reply:
done